verify-signatures.markdown

```diff
@@ -36,6 +36,40 @@ To verify signature run in the terminal:
 jarsigner -verify -verbose <path_to_jar>
 ```
 
+### <a name="WindowsSigner">Windows</a>
+ODBC driver .msi can be verified by a few different methods.
+
+Verify signature in PowerShell version > 5.1:
+```
+Get-AuthenticodeSignature -FilePath <path_to_msi>
+```
+
+Verify signature using SigCheck:
+
+- Download [SigCheck](https://learn.microsoft.com/en-us/sysinternals/downloads/sigcheck)
+
+Verify signature using SignTool:
+
+- Download [SignTool](https://learn.microsoft.com/en-us/windows/win32/seccrypto/using-signtool-to-verify-a-file-signature)
+
+Signature Certificate:
+```
+2DA2 DC02 8EE6 42CD 77C4 BA04 F289 1F24 7831 2C29
+```
+
+### <a name="CodeSigner">CodeSign</a>
+ODBC driver .pkg is signed using CodeSign.
+
+To verify signature run in the terminal:
+```
+pkgutil --verbose --check-signature <path_to_pkg>
+```
+
+Certificate Fingerprint:
+```
+49 68 39 4A BA 83 3B F0 CC 5E 98 3B E7 C1 72 AC 85 97 65 18 B9 4C BA 34 62 BF E9 23 76 98 C5 DA
```

Comment:
is this downloadable somewhere?

Reply:
I haven't been able to find them anywhere. If anyone knows where I could plz link!